src/site/apt/usage.apt.vm

```diff
@@ -688,6 +688,24 @@ mvn dependency:build-classpath -Dmdep.outputFile=cp.txt
   poms and declared in transitive dependency poms.
 
 
+* <<<dependency:list-repositories>>>
+
+  This goal is used to list all the plugin repositories that this build depends upon. It will show plugin repositories defined in your settings,
```

Comment:
is used to list --> lists
will show --> shows

Reply:
Implemented correct English...